src/Symfony/Component/HttpKernel/Fragment/HIncludeFragmentRenderer.php

```diff
@@ -29,41 +28,29 @@ class HIncludeFragmentRenderer extends RoutableFragmentRenderer
 {
     private $globalDefaultTemplate;
     private $signer;
-    private $templating;
+    private $twig;
     private $charset;
 
     /**
-     * @param EngineInterface|Environment $templating            An EngineInterface or a Twig instance
+     * @param Environment|null            $twig                  A Twig instance
```

Comment:
the phpdoc does not seem useful. afaik we don't add phpdocs for params if they don't add any value